packages/core/aqua-to-js-compiler/package.json

```diff
@@ -0,0 +1,28 @@
+{
+    "name": "@fluencelabs/aqua-to-js-compiler",
+    "type": "module",
+    "version": "0.0.0",
+    "description": "Tool for generating aqua wrapper",
+    "main": "dist/index.js",
+    "scripts": {
+        "test": "vitest run",
+        "build": "tsc"
+    },
+    "keywords": [],
+    "author": "Fluence Labs",
+    "license": "Apache-2.0",
+    "dependencies": {
+        "@fluencelabs/aqua-api": "0.12.0",
+        "@fluencelabs/aqua-lib": "0.7.3",
+        "@fluencelabs/interfaces": "workspace:*",
+        "@fluencelabs/js-client": "*",
+        "@fluencelabs/registry": "0.8.7",
+        "@fluencelabs/spell": "0.5.20",
+        "@fluencelabs/trust-graph": "0.4.7",
+        "ts-pattern": "5.0.5"
```

Comment:
I believe you don't need the following stuff in direct dependencies of this library
@fluencelabs/aqua-lib
@fluencelabs/registry
@fluencelabs/spell
@fluencelabs/trust-graph

Reply:
Fixed